sdk/healthcareapis/Microsoft.Azure.Management.HealthcareApis/src/Generated/IPrivateEndpointConnectionsOperations.cs

```diff
@@ -47,7 +47,7 @@ public partial interface IPrivateEndpointConnectionsOperations
         /// <exception cref="Microsoft.Rest.ValidationException">
         /// Thrown when a required parameter is null
         /// </exception>
-        Task<AzureOperationResponse<IEnumerable<PrivateEndpointConnection>>> ListByServiceWithHttpMessagesAsync(string resourceGroupName, string resourceName, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
+        Task<AzureOperationResponse<IEnumerable<PrivateEndpointConnectionDescription>>> ListByServiceWithHttpMessagesAsync(string resourceGroupName, string resourceName, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
This name doesn't seem ideal given that "Description" typically refers to a human friendly sentence which describes something.  It appears all we wanted to do was add SystemData to an existing model, can we explore adding this directly to the PrivateEndpointConnection object instead of creating a new type?

Reply:
In my understanding, this code is generated automatically, change the code will cause the error in CI.